### DISCUSSION TITLE
Relative import paths need explicit file extensions in ECMAScript

### DISCUSSION THREAD

**Discussion by LWJerri** [scalar/scalar#2887] [Q&A] [answered]
Hello, dear community! Today I bumped a package version and now I an error:

![image](https://github.com/user-attachments/assets/5f05379c-0af3-47e1-9a87-2ba3b5d7dc06)

`package.json`
```json
{
  ...another things
  "@scalar/nestjs-api-reference": "0.3.139"
}
```

`tsconfig.json`
```json
{
  "compilerOptions": {
    "module": "NodeNext",
    "declaration": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "moduleResolution": "NodeNext",
    "target": "ES2021",
    "sourceMap": true,
    "outDir": "dist",
    "baseUrl": "./",
    "allowJs": true,
    "incremental": true,
    "skipLibCheck": true,
    "esModuleInterop": true,
    "noImplicitAny": false,
    "strictNullChecks": false,
    "noUnusedParameters": true,
    "strictBindCallApply": false,
    "forceConsistentCasingInFileNames": false,
    "noFallthroughCasesInSwitch": false
  }
}
```

**Comment by amritk** [comment]
We have a fix coming out

https://github.com/scalar/scalar/pull/2891

Thanks for the report!

**Comment by LWJerri** [comment]
@amritk, just updated package version and problem is still there. Already tried to delete `node_modules` folder.

**Reply by amritk** [reply, marked as answer]
sorry @LWJerri the PR is merged but the fix is still deploying, it will be in `@scalar/nestjs-api-reference@0.3.141`

**Reply by LWJerri** [reply]
> sorry @LWJerri the PR is merged but the fix is still deploying, it will be in `@scalar/nestjs-api-reference@0.3.141`

Well, nice. Thanks for ur job!
